modules/data_release/ajax/FileUpload.php

```diff
@@ -15,11 +15,11 @@
 $DB   = \Database::singleton();
 $user = \User::singleton();
 
-if ($_POST['action'] == 'upload'
+if ($_GET['action'] == 'upload'
     && $user->hasPermission("data_release_upload")
 ) {
     $fileName    = $_FILES["file"]["name"];
-    $version     = $_POST['version'];
+    $version     = $_POST['version'] ? $_POST['version'] : null;
```

Comment:
Could you make this check more explicit, e.g. if it's supposed to be not empty, or an int, etc.?

This statement will for example change a value of `0` to null. Maybe this is correct behaviour but it's hard to know.